pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -188,7 +188,7 @@ public void validate(final TokenCredentials credentials, final WebContext contex
           	createJwtProfile(credentials, jwt);
 
         } catch (final ParseException e) {
-            throw new TechnicalException("Cannot decrypt / verify JWT", e);
+            throw new CredentialsException("Cannot decrypt / verify JWT", e);
```

Comment:
OK. This changes the meaning: unparseable JWT are now considered just as normal credentials validation failures as others.